src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use './variables' as *;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the image path is correct and that the image file exists at the specified location within the project structure. If the path is incorrect, it will result in the star image not being displayed.